source/libs/page-detect.ts

```diff
@@ -222,11 +222,7 @@ export const _isQuickPR = [
 	'https://github.com/sindresorhus/refined-github/compare/test-branch?quick_pull=1'
 ];
 
-export const isReleasesOrTags = (): boolean => {
-	const parts = (getRepoPath() ?? '').split('/');
-	return /^(releases|tags)$/.test(parts[0]) && parts[1] !== 'new';
-};
-
+export const isReleasesOrTags = (): boolean => /^tags$|^releases($|\/tag)/.test(getRepoPath()!);
```

Comment:
```suggestion
export const isReleasesOrTags = (): boolean => ['tags', 'releases', 'releases/tag'].includes(getRepoPath()!);
```

Reply:
`includes` is a direct match so it won't catch `releases/tags/1.2.3`

`.startsWith` also can't be used because `releases` should match but `releases/new` shouldn't